README.md

```diff
@@ -1,11 +1,36 @@
-Apigee Edge Drupal module
----
+# Apigee Edge Drupal module
 
-The Apigee Edge module enables you to integrate Drupal 8 with Apigee Edge.
+The Apigee Edge module enables you to integrate Drupal 8 with Apigee Edge.  
 
-**Note**: The Apigee Edge module requires Drupal 8.6.x or higher and PHP 7.1 or higher.
+* Users that register or are added to the Drupal site will be added as developers in Apigee Edge
+* Click on "My Apps" in user menu to get keys for Apigee Edge APIs
```

Comment:
I would have thought `My Apps` was for consumers to get their keys for the provider's APIs. Is this also how you get your keys for Apigee Edge APIs?

Reply:
Yes, updated